src/FSLibrary/StellarKubeSpecs.fs

```diff
@@ -836,13 +836,37 @@ type NetworkCfg with
         let historyBackend (pn: PodName) : Extensionsv1beta1IngressBackend =
             Extensionsv1beta1IngressBackend(serviceName = pn.StringName, servicePort = IntstrIntOrString(value = "80"))
 
+        let isNginx =
+            self.missionContext.ingressClass = "nginx-private"
```

Comment:
Since, we are planning to merge this change after we've migrated existing ingress class from `nginx-private` to `private`. We should keep this as `private`.